content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,331 @@ Download the Istio release which includes installation files, samples, and the
     $ cd istio-{{< istio_full_version >}}
     {{< /text >}}
 
-    The installation directory contains:
+1.  Add the `istioctl` client to your path (Linux or macOS):
 
-    - Installation YAML files for Kubernetes in `install/kubernetes`
-    - Sample applications in `samples/`
-    - The [`istioctl`](/docs/reference/commands/istioctl) client binary in the `bin/` directory. `istioctl` is
-      used when manually injecting Envoy as a sidecar proxy.
```

Comment:
Like I said above, these instructions are pointed to by all the more advanced install options, so this is pretty useful information that we probably don't want to lose.

Reply:
fixed